resources/public/api/conf/3.0/schemas/retrieve_property_bsas.json

```diff
@@ -1,154 +1,675 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Retrieve a UK property Business Source Adjustable Summary Response",
+  "description": "Retrieve a UK property Business Source Adjustable Summary Response.",
   "type": "object",
-  "required": [
-    "metadata"
-  ],
   "properties": {
     "metadata": {
-      "$ref": "metadata_definitions.json#/definitions/propMetadata"
+      "type": "object",
+      "description": "Object containing the identifying data for the business and associated information for the BSAS with this ID.",
+      "properties": {
+        "calculationId": {
+          "description": "The unique identifier of the summary calculation.",
+          "type": "string",
+          "example": "717f3a7a-db8e-11e9-8a34-2a2ae2dbcce4"
+        },
+        "requestedDateTime": {
+          "description": "The date and time the summary calculation was originally requested, prior to any adjustments.",
+          "type": "string",
+          "example": "2019-04-04T01:01:01Z"
+        },
+        "adjustedDateTime": {
+          "description": "The date and time this calculation was adjusted.",
+          "type": "string",
+          "example": "2019-04-04T01:01:01Z"
+        },
+        "nino": {
+          "description": "National Insurance number, in the format AA999999A.",
+          "type": "string",
+          "example": "TC663795B"
+        },
+        "taxYear": {
+          "description": "The tax year the data applies to, for example, 2021-22. The start year and end year must not span two tax years. No gaps are allowed, for example, 2020-22 is not valid.",
+          "type": "string",
+          "example": "2021-22"
+        },
+        "summaryStatus": {
+          "type": "string",
+          "enum": [
+            "valid",
+            "invalid",
+            "superseded"
+          ],
+          "description": "The status of the this calculated summary, limited to the following possible values: \n\n valid \n\n invalid \n\n superseded"
```

Comment:
`limited to the following possible values: \n\n valid \n\n invalid \n\n superseded` needs to be removed from the documentation as  it is duplicating on the documentation